daal4py/sklearn/ensemble/GBTDAAL.py

```diff
@@ -0,0 +1,306 @@
+# *******************************************************************************
+# Copyright 2014-2020 Intel Corporation
+# All Rights Reserved.
+#
+# This software is licensed under the Apache License, Version 2.0 (the
+# "License"), the following terms apply:
+#
+# You may not use this file except in compliance with the License.  You may
+# obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# *******************************************************************************
+
+# daal4py GBT scikit-learn-compatible estimator class
+
+import numpy as np
+import numbers
+import warnings
+from sklearn.base import BaseEstimator, ClassifierMixin, RegressorMixin
+from sklearn.utils.validation import check_X_y, check_array, check_is_fitted
+from sklearn import preprocessing
+from sklearn.utils.multiclass import check_classification_targets
+from sklearn.utils import check_random_state
+import daal4py as d4p
+from ..utils import getFPType
+
+from sklearn import __version__ as sklearn_version
+from distutils.version import LooseVersion
+
+daal_version = tuple(map(int, (d4p.__daal_link_version__[0:4], d4p.__daal_link_version__[4:8])))
```

Comment:
Perhaps we should use `d4py.__daal_run_version__` instead?

Reply:
Yes, you are right. Thanks!